src/Renderer/c3DEngine.js

```diff
@@ -15,7 +15,7 @@ import RendererConstant from './RendererConstant';
 
 var instance3DEngine = null;
 
-function c3DEngine(scene, positionCamera, viewerDiv, debugMode, gLDebug) {
+function c3DEngine(scene, positionCamera, targetCamera, viewerDiv, debugMode, gLDebug) {
```

Comment:
This is specific to GlobeControls, so to a Globe. I would prefer not to add that as a parameter to c3DEngine.